api/repositories.py

```diff
@@ -29,6 +29,14 @@ class UserRepository(Repository):
     def __init__(self, mongo_collection, repository_collection):
         Repository.__init__(self, mongo_collection, repository_collection)
 
+    def update_user_account(self,user_id,name,password,house_name,house_location):
+        user = self.get_user_by_id(user_id)
+        if user is None:
+            return None
+        self.collection.update_one({'user_id': user_id, 'name': name,
```

Comment:
How do you accommodate the requirement that None-values don't affect the current value?